### PR TITLE
Update API docs

### PR DESCRIPTION
**Pull request by ayeshLK**
## Examples
N/A

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/commons.bal [hunk 1]
```diff
@@ -182,10 +150,20 @@ type RequestQueryParams record {|
     string hubReason?;
 |};
 
-# Common Responses to be used in the subscriber-service implementation.
+# Common response which could be used for `websub:Acknowledgement`.
```
Comment (praneesha) [inline ballerina/commons.bal:153]:
```suggestion
# Common response, which could be used for the `websub:Acknowledgement`.
```

Comment (praneesha) [reply ballerina/commons.bal:153]: Please fix all occurrences.

File: ballerina/commons.bal [hunk 2]
```diff
@@ -182,10 +150,20 @@ type RequestQueryParams record {|
...
 public final readonly & Acknowledgement ACKNOWLEDGEMENT = {};
+
+# Common response which could be used for `websub:SubscriptionVerificationSuccess`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:156]:
```suggestion
# Common response, which could be used for `websub:SubscriptionVerificationSuccess`.
```

File: ballerina/commons.bal [hunk 3]
```diff
@@ -182,10 +150,20 @@ type RequestQueryParams record {|
...
 public final readonly & SubscriptionVerificationSuccess SUBSCRIPTION_VERIFICATION_SUCCESS = {};
+
+# Common response which could be used for `websub:SubscriptionVerificationError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:159]:
```suggestion
# Common response, which could be used for `websub:SubscriptionVerificationError`.
```

File: ballerina/commons.bal [hunk 4]
```diff
@@ -182,10 +150,20 @@ type RequestQueryParams record {|
...
 public final SubscriptionVerificationError SUBSCRIPTION_VERIFICATION_ERROR = error SubscriptionVerificationError("Subscription verification failed");
+
+# Common response which could be used for `websub:UnsubscriptionVerificationSuccess`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:162]:
```suggestion
# Common response, which could be used for `websub:UnsubscriptionVerificationSuccess`.
```

File: ballerina/commons.bal [hunk 5]
```diff
@@ -182,10 +150,20 @@ type RequestQueryParams record {|
...
 public final readonly & UnsubscriptionVerificationSuccess UNSUBSCRIPTION_VERIFICATION_SUCCESS = {};
+
+# Common response which could be used for `websub:UnsubscriptionVerificationError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:165]:
```suggestion
# Common response, which could be used for `websub:UnsubscriptionVerificationError`.
```

File: ballerina/commons.bal [hunk 6]
```diff
@@ -182,10 +150,20 @@ type RequestQueryParams record {|
...
 public final UnsubscriptionVerificationError UNSUBSCRIPTION_VERIFICATION_ERROR = error UnsubscriptionVerificationError("Unsubscription verification failed");
+
+# Common response which could be used for `websub:SubscriptionDeletedError`.
```
Comment (ayeshLK) [inline ballerina/commons.bal:168]:
```suggestion
# Common response, which could be used for `websub:SubscriptionDeletedError`.
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**